contracts/cryptoblades.sol

```diff
@@ -390,7 +390,7 @@ contract CryptoBlades is Initializable, AccessControlUpgradeable {
     function mintCharacter() public onlyNonContract doesNotHaveMoreThanMaxCharacters oncePerBlock(msg.sender) requestPayFromPlayer(mintCharacterFee) {
```

Comment:
`requestPayFromPlayer` modifier is going to check against in-game-only SKILL and unclaimed SKILL still.